<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1476462003437" stop="1476462012655" version="1.4.16" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>Test-class Suite : Test-class test</name>
    <title>Test-class Suite : Test-class test</title>
    <test-cases>
        <test-case start="1476462003974" stop="1476462003986" status="pending">
            <name>main</name>
            <failure>
                <message>Test not implemented yet</message>
                <stack-trace>There are no stack trace</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="story" value="Story1"/>
                <label name="story" value="Story2"/>
                <label name="feature" value="My Feature"/>
                <label name="host" value="Aams-MacBook-Pro-2.local"/>
                <label name="thread" value="14326@Aams-MacBook-Pro-2.local.main(1)"/>
            </labels>
        </test-case>
        <test-case start="1476462004016" stop="1476462012653" status="broken">
            <name>allElements</name>
            <failure>
                <message>StaleElementReferenceException: Element is no longer attached to the DOM
Command duration or timeout: 5 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'Aams-MacBook-Pro-2.local', ip: '192.168.59.3', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.6', java.version: '1.8.0_92'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=33.0, platform=MAC, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 5ceb9f94-41d1-7c48-892a-b9980369148f</message>
                <stack-trace>org.openqa.selenium.StaleElementReferenceException: Element is no longer attached to the DOM
Command duration or timeout: 5 milliseconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'Aams-MacBook-Pro-2.local', ip: '192.168.59.3', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.6', java.version: '1.8.0_92'
Driver info: org.openqa.selenium.firefox.FirefoxDriver
Capabilities [{applicationCacheEnabled=true, rotatable=false, handlesAlerts=true, databaseEnabled=true, version=33.0, platform=MAC, nativeEvents=false, acceptSslCerts=true, webStorageEnabled=true, locationContextEnabled=true, browserName=firefox, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true}]
Session ID: 5ceb9f94-41d1-7c48-892a-b9980369148f
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:206)
	at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:678)
	at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:327)
	at org.openqa.selenium.remote.RemoteWebElement.getText(RemoteWebElement.java:179)
	at spreeuitest.ElimentsTest.allElements(ElimentsTest.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: org.openqa.selenium.StaleElementReferenceException: Element is no longer attached to the DOM
For documentation on this error, please visit: http://seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '2.53.0', revision: '35ae25b1534ae328c771e0856c93e187490ca824', time: '2016-03-15 10:43:46'
System info: host: 'Aams-MacBook-Pro-2.local', ip: '192.168.59.3', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.6', java.version: '1.8.0_92'
Driver info: driver.version: unknown
	at &lt;anonymous class&gt;.fxdriver.cache.getElementAt(resource://fxdriver/modules/web-element-cache.js:9457)
	at &lt;anonymous class&gt;.Utils.getElementAt(file:///var/folders/zd/169rjdb53qv_dfmhsbk47c1c0000gn/T/anonymous5865551040773119957webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:9039)
	at &lt;anonymous class&gt;.WebElement.getElementText(file:///var/folders/zd/169rjdb53qv_dfmhsbk47c1c0000gn/T/anonymous5865551040773119957webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12092)
	at &lt;anonymous class&gt;.DelayedCommand.prototype.executeInternal_/h(file:///var/folders/zd/169rjdb53qv_dfmhsbk47c1c0000gn/T/anonymous5865551040773119957webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12661)
	at &lt;anonymous class&gt;.DelayedCommand.prototype.executeInternal_(file:///var/folders/zd/169rjdb53qv_dfmhsbk47c1c0000gn/T/anonymous5865551040773119957webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12666)
	at &lt;anonymous class&gt;.DelayedCommand.prototype.execute/&lt;(file:///var/folders/zd/169rjdb53qv_dfmhsbk47c1c0000gn/T/anonymous5865551040773119957webdriver-profile/extensions/fxdriver@googlecode.com/components/command-processor.js:12608)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="Aams-MacBook-Pro-2.local"/>
                <label name="thread" value="14326@Aams-MacBook-Pro-2.local.pool-1-thread-1(13)"/>
            </labels>
        </test-case>
        <test-case start="1476462005510" stop="1476462005660" status="broken">
            <name>HomeTest</name>
            <failure>
                <message>Exception: null</message>
                <stack-trace>java.lang.Exception
	at utility.Utils.takeScreenshot(Utils.java:113)
	at spreeuitest.HomeTest.HomeTest(HomeTest.java:90)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="story" value="W"/>
                <label name="story" value="Story2"/>
                <label name="feature" value="My Feature"/>
                <label name="host" value="Aams-MacBook-Pro-2.local"/>
                <label name="thread" value="14326@Aams-MacBook-Pro-2.local.pool-1-thread-2(14)"/>
            </labels>
        </test-case>
        <test-case start="1476462005664" stop="1476462005664" status="broken">
            <name>afterMethod</name>
            <description>@AfterMethod</description>
            <failure>
                <message>NullPointerException: null</message>
                <stack-trace>java.lang.NullPointerException
	at spreeuitest.beforeaftertest.afterMethod(beforeaftertest.java:71)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:514)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:215)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:707)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="Aams-MacBook-Pro-2.local"/>
                <label name="thread" value="14326@Aams-MacBook-Pro-2.local.pool-1-thread-2(14)"/>
            </labels>
        </test-case>
        <test-case start="1476462005667" stop="1476462005668" status="canceled">
            <name>beforeMethod</name>
            <description>@BeforeMethod</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:366)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onConfigurationSkip(AllureTestListener.java:126)
	at org.testng.internal.Invoker.runConfigurationListeners(Invoker.java:1661)
	at org.testng.internal.Invoker.handleConfigurationSkip(Invoker.java:257)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:195)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:589)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="host" value="Aams-MacBook-Pro-2.local"/>
                <label name="thread" value="14326@Aams-MacBook-Pro-2.local.pool-1-thread-2(14)"/>
            </labels>
        </test-case>
        <test-case start="1476462005668" stop="1476462005668" status="canceled">
            <name>accountButtonVerification</name>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:366)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:181)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1683)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1675)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:700)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="story" value="Story1"/>
                <label name="story" value="Story2"/>
                <label name="feature" value="My Feature"/>
                <label name="host" value="Aams-MacBook-Pro-2.local"/>
                <label name="thread" value="14326@Aams-MacBook-Pro-2.local.pool-1-thread-2(14)"/>
            </labels>
        </test-case>
        <test-case start="1476462005669" stop="1476462005670" status="canceled">
            <name>accountButtonVerification</name>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:366)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:181)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1683)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1675)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:700)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="story" value="Story1"/>
                <label name="story" value="Story2"/>
                <label name="feature" value="My Feature"/>
                <label name="host" value="Aams-MacBook-Pro-2.local"/>
                <label name="thread" value="14326@Aams-MacBook-Pro-2.local.pool-1-thread-2(14)"/>
            </labels>
        </test-case>
        <test-case start="1476462005671" stop="1476462005671" status="canceled">
            <name>Product1Test</name>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:366)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:181)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1683)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1675)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:700)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="story" value="Ruby on Rail Tote"/>
                <label name="story" value="Ruby on Rail Begs"/>
                <label name="feature" value="Verify Product Information"/>
                <label name="host" value="Aams-MacBook-Pro-2.local"/>
                <label name="thread" value="14326@Aams-MacBook-Pro-2.local.pool-1-thread-2(14)"/>
            </labels>
        </test-case>
        <test-case start="1476462005671" stop="1476462005672" status="canceled">
            <name>ProductSelectionTest</name>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:366)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:181)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1683)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1675)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:700)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="story" value="Story1"/>
                <label name="story" value="Story2"/>
                <label name="feature" value="My Feature"/>
                <label name="host" value="Aams-MacBook-Pro-2.local"/>
                <label name="thread" value="14326@Aams-MacBook-Pro-2.local.pool-1-thread-2(14)"/>
            </labels>
        </test-case>
        <test-case start="1476462005673" stop="1476462005673" status="canceled">
            <name>BrowserTest</name>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:366)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:181)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1683)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1675)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:700)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="story" value="waseyrabby@btinternet.com"/>
                <label name="story" value="password"/>
                <label name="feature" value="SignIN TO Spree Account"/>
                <label name="host" value="Aams-MacBook-Pro-2.local"/>
                <label name="thread" value="14326@Aams-MacBook-Pro-2.local.pool-1-thread-2(14)"/>
            </labels>
        </test-case>
        <test-case start="1476462005674" stop="1476462005674" status="canceled">
            <name>signInTest</name>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:366)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:181)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1683)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1675)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:700)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="story" value="Story1"/>
                <label name="story" value="Story2"/>
                <label name="feature" value="SignIN TO Spree Account"/>
                <label name="host" value="Aams-MacBook-Pro-2.local"/>
                <label name="thread" value="14326@Aams-MacBook-Pro-2.local.pool-1-thread-2(14)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
